db/migrate/20171024183226_create_books.rb

```diff
@@ -2,6 +2,8 @@ class CreateBooks < ActiveRecord::Migration[5.1]
   def change
     create_table :books do |t|
       t.string :title
+
+      t.timestamps
```

Comment:
Note that you don't want to go back and change old migrations. Instead make a new migration to implement changes.